man/git-extras.md

```diff
@@ -51,7 +51,7 @@ git-extras(1) -- Awesome GIT utilities
    - **git-delete-tag(1)** Delete tags
    - **git-delta(1)** Lists changed files
    - **git-effort(1)** Show effort statistics on file(s)
-   - **git-feature(1)** Create/Merge feature branch
+   - **# git-feature(1)** Create/Merge feature branch
```

Comment:
Look like we need to make the title of `man/git-feature.md` the same as other files.

Reply:
yep - this was auto generated.